test/e2e/snaps/test-snap-bip-44.spec.ts

```diff
@@ -28,11 +28,9 @@ describe('Test Snap bip-44', function () {
         const testSnaps = new TestSnaps(driver);
 
         // Navigate to `test-snaps` page, and install the Snap.
-        await openTestSnapClickButtonAndInstall(
-          driver,
-          'connectBip44Button',
-          true,
-        );
```

Comment:
with the objects for options, now it's easier to understand the option we are enabling